.vscode/launch.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
I did include this on purpose, because it's really useful when you're trying to debug the tests in VS Code. However, since it's tool specific, I'll remove it.